src/pages/forgotPassword/ResetPassword.tsx

```diff
@@ -0,0 +1,65 @@
+import { FunctionComponent, useState } from "react";
+import { FormikValues, useFormikContext } from "formik";
+import { Box, Stack, useTheme } from "@mui/material";
+import { Button, Typography } from "elements";
+import { PasswordInputField } from "components";
+
+const ResetPassword: FunctionComponent = () => {
+  const theme = useTheme();
+  const [maskPassword, setMaskPassword] = useState(true);
+  const {
+    isValid,
+    values: { confirmPassword, newPassword },
+  } = useFormikContext<FormikValues>();
+
+  const togglePasswordMask = () => {
+    setMaskPassword(!maskPassword);
+  };
```

Comment:
Is this considered a duplicate or just added as safety?` PasswordInputField.tsx` has the following as default:
```
  const togglePasswordMask = () => {
    setInternalMaskPassword(!internalMaskPassword);
  };
  ...
  
  onClick={ handlePressEndAdornment || togglePasswordMask }
```